### PR TITLE
Actual fix

### PR DESCRIPTION
**Pull request by partouf**

### CODE REVIEW

File: src/objdump/parser.cpp [hunk 1]
```diff
@@ -29,6 +29,15 @@ AsmParser::ObjDumpParser::ObjDumpParser(const Filter &filter) : filter(filter)
     reproducible = false;
 }
 
+void AsmParser::ObjDumpParser::updateSourceRefLineNumber()
+{
+    auto lineNum = atoi(this->state.text.c_str());
```
Comment (jeremy-rifkin) [inline src/objdump/parser.cpp:34]: Is source file name handled anywhere?

Comment (partouf) [reply src/objdump/parser.cpp:34]: Yes, line 410

Comment (jeremy-rifkin) [reply src/objdump/parser.cpp:34]: Ah I see, thanks!

**Review by jeremy-rifkin — APPROVED** [review]

**Comment by partouf** [comment]
oh, I just realized this will go wrong when paths have spaces

**Pull request merged**